addons/account/i18n/account.pot

```diff
@@ -15551,6 +15554,11 @@ msgstr ""
 msgid "tax"
 msgstr ""
 
+#. module: account
+#: model:ir.model.fields,field_description:account.field_account_tax_group__property_advance_tax_payment_account_id
+msgid "Tax Advance Account"
+msgstr ""
+
```

Comment:
Why are you touching the translations ? I don't think they already have been generated for 16.0 ^^ (traditionally, this is a last-minute action). Plus, you're even removing one, and this wouldn't be a good idea in stable :p